events-src/Google.Events.SystemTextJson/Cloud/PubSub/V1/PubsubMessageData.g.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright 2020, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+// Note: we'd expect to generate this type from a JSON schema.
+
+namespace Google.Events.SystemTextJson.Cloud.PubSub.V1
+{
+    /// <summary>
+    /// The CloudEvent representation of a PubSub message.
+    /// </summary>
+    public sealed partial class PubsubMessageData
```

Comment:
Should this be called Pub**S**ubMessageData. The namespace is the file is not. There was a similar comment on the protobuf definition on yesterday's PR.

Reply:
Yup, which I would have commented on if I hadn't moved it :)
Basically, this mirrors what's in the API itself: https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto#L188
(C# namespace includes PubSub, message is PubsubMessage.)

So I'm being consistent with the API, basically. But I agree it's annoying.